tensorboard/uploader/proto/export_service.proto

```diff
@@ -15,6 +16,8 @@ service TensorBoardExporterService {
   // Stream scalars for all the runs and tags in an experiment.
   rpc StreamExperimentData(StreamExperimentDataRequest)
       returns (stream StreamExperimentDataResponse) {}
+  // Stream BLOB as chunks for a given blob_id.
```

Comment:
Please downcase “blob” as in the rest of TensorBoard and Blobstore.
(I recognize that it’s sometimes taken to stand for “binary large
object”, but we don’t use it that way.)

Reply:
Done.